features/old/applications/providers/create_application.feature

```diff
@@ -1,4 +1,4 @@
-@ignore-backend
+@javascript @ignore-backend
```

Comment:
The rule of thumb. When touching old cucumber, we migrate to the new way

Reply:
WDYM the new way?